packages/cli/src/utils/config.js

```diff
@@ -1,67 +1,20 @@
 import path from 'path'
-import consola from 'consola'
 import defaultsDeep from 'lodash/defaultsDeep'
-import { defaultNuxtConfigFile, getDefaultNuxtConfig } from '@nuxt/config'
-import { clearRequireCache, scanRequireTree } from '@nuxt/utils'
-import esm from 'esm'
+import { loadNuxtConfig as _loadNuxtConfig, getDefaultNuxtConfig } from '@nuxt/config'
```

Comment:
Will `@nuxt/config loadNuxtConfig` be confused with `@nuxt/cli loadNuxtConfig`, maybe change `@nuxt/config loadNuxtConfig` sth else ?

Reply:
Actually `loadNuxtConfig` from `@nuxt/cli` should be used only for 

I would rather not export `loadNuxConfig` from `@nuxt/cli` anymore and rename it to `loadNuxtConfigFromArgv(argv, configContext)`, might be a really rare breaking change.

Reply:
Indeed i considered both points. To avoid breaking changes of `@nuxt/cli` package, export is there but it is not even documented so should be fine. New `loadNuxtConfig` export is exposed by core to distro but i think for now we should only document `loadNuxt`

Reply:
Agree :)